src/app/pages/send/ordinal-inscription/send-inscription-review.tsx

```diff
@@ -40,8 +43,19 @@ export function SendInscriptionReview() {
   const { refetch } = useCurrentNativeSegwitUtxos();
   const { broadcastTx, isBroadcasting } = useBitcoinBroadcastTransaction();
 
+  // Filer out inscription txid from the check list
+  const filteredInputs = useMemo(() => {
+    const decodedTx = decodeBitcoinTx(bytesToHex(signedTx));
+    return decodedTx.inputs.filter(input => inscription.tx_id !== bytesToHex(input.txid));
```

Comment:
What's happening here?

Reply:
here we filter out inscription txid that we are sending from checking for inscribed utxo

Reply:
if we don't filter it out, we won't be able to send tx, as check will always fail for inscription send flow

Reply:
Got it. Makes sense, just wonder if we can just present code as some functions to make it more readable?

`function filterOutIntentionalInscriptionSpend() {}`

This would make it so much easier to understand

Reply:
fixed